operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -160,6 +165,36 @@ func (c *CentralComponentSpec) GetAdminPasswordGenerationDisabled() bool {
 	return pointer.BoolPtrDerefOr(c.AdminPasswordGenerationDisabled, false)
 }
 
+// Monitoring defines settings for monitoring endpoint.
+type Monitoring struct {
+	// Exposes monitoring endpoint by adding port 9090 with the name
+	// "monitoring" to the deployment, creating service with the same port,
+	// and adding the required network policy.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=1
```

Comment:
Just as a suggestion from my side, in which direction we should move with the comment, let's see what other's are suggesting:

```suggestion
	// Expose Central's monitoring endpoint. A new service, "monitoring", with port
	// 9090 will be created as well as a network policy allowing inbound connections
	// to the port.
	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=1
```